doc/developer/debugging.rst

```diff
@@ -5,17 +5,23 @@ Application debugging
 
 The goal of this document is to give some troubleshooting tips.
 
+At first you should copy the file ``docker-compose.override.sample.yaml`` to
+``docker-compose.override.yaml``, and update the composition ``docker-compose up -d``.
```

Comment:
not sure what we mean with "update the composition". Do we simply mean restart the application?
So
docker-compose down
docker-compose up -d

Reply:
In this case ``docker-compose up -d`` is enough :-)